From 11ec3593f677e43da49a9fb85fdda5538103c731 Mon Sep 17 00:00:00 2001
From: Martin Magr <mmagr@redhat.com>
Date: Fri, 25 Apr 2014 18:05:02 +0200
Subject: [PATCH] RHEL should have default provider

On RHEL-7 systemd is used and the default provider (service I suppose)
works on older releases too.

Change-Id: I4cb378350cb8de7f7c1ce1c3caacabbe682227da
---
 manifests/params.pp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/manifests/params.pp b/manifests/params.pp
index d20d337..c3c8456 100644
--- a/manifests/params.pp
+++ b/manifests/params.pp
@@ -39,7 +39,7 @@ class nova::params {
       $root_helper                  = 'sudo nova-rootwrap'
       $lock_path                    = '/var/lib/nova/tmp'
       case $::operatingsystem {
-        'Fedora': {
+        'Fedora', 'RedHat': {
           $special_service_provider = undef
         }
         default: {
-- 
1.8.5.3

